images/test-environments/linux/Dockerfile

```
@@ -0,0 +1,34 @@
+ARG CONDA_VER=latest
+ARG UBUNTU_VER=20.04
+ARG OS_TYPE=x86_64
+ARG PY_VER=3.7
+
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -yq curl wget jq
+
+ARG CONDA_VER
+ARG OS_TYPE
+# Install miniconda to /miniconda
+RUN curl -LO "http://repo.continuum.io/miniconda/Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh"
+RUN bash Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh -p /miniconda -b
+RUN rm Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh
```

Combining these three steps in a single layer can significantly reduce layer sizes (and hence the size of the final image) by preventing temporary files that are removed in later steps from being committed as layers. That can help speed up pipelines that depend on this image by reducing the amount that those builds need to download.
```suggestion
RUN curl -LO "http://repo.continuum.io/miniconda/Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh" && \
    bash Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh -p /miniconda -b && \
    rm Miniconda3-${CONDA_VER}-Linux-${OS_TYPE}.sh
```